packages/core/src/command.tsx

```diff
@@ -164,14 +165,34 @@ export function apply(ctx: Context, config: Config) {
                   })()}
                   src={url}
                 ></img>
-              </message>,
+              </>,
             )
         }
+        output.push(children)
+      }
+
+      switch (session.resolve(config.outputMethod)) {
+        case 'one-by-one':
+          return output.map((children) => <message>{children}</message>)
+        case 'merge-multiple':
+          return <message>{output.map((children) => children)}</message>
+        case 'forward-all':
+          return (
+            <message forward>
+              {output.map((children) => (
+                <message>{children}</message>
```

Comment:
```suggestion
                <message id="{session.bot.selfId}">{children}</message>
```

See: https://koishi.chat/zh-CN/api/message/elements.html#message